lci_strategic_plugin/config/parameters.yaml

```diff
@@ -35,6 +35,15 @@ desired_distance_to_stop_buffer : 15.0
 # Double: The minimum period in seconds which a maneuver plan must cover if the plugin wishes to control the whole plan
 min_maneuver_planning_period : 15.1
 
+# Double: Update time interval of carma streets. Units in s
+delta_t : 1.0
```

Comment:
This variable should have a more descriptive name. Like carma_streets_update_interval or something. delta_t is particularly confusing in a node containing a bunch of kinematic calculations